### PR TITLE
component: taost and snackbar

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Added Toast Basic which can be use as snackbar too

### CODE REVIEW

**Comment by ShrutiLakhani** [comment]
Looks good to me!!!

File: styles/global.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
   --danger-color: #d00d65;
   --warning-color: #f5a623;
   --success-color: #17c964;
+
   --info-color: #03a9f4;
   --white-color: #ffff;
   --border-color: rgb(221, 221, 221);
```
Comment (notmafiaa) [inline styles/global.css:11]: color variables can be improved. for eg --COLOR-RED, capitalize naming can be easily seen. through --COLOR you can access the whole list of colors in VSCODE.

File: styles/toast.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+.toast > button {
+  all: unset;
+  cursor: pointer;
+  display: flex;
+  margin-left: var(--spacing-sm);
+}
+
+.toast-success {
+  background-color: var(--success-color);
```
Comment (notmafiaa) [inline styles/toast.css:21]: good naming conventions for classes.

specificity check is also good.

**Review by notmafiaa — APPROVED** [review]
rest of the code looks great.

have commented where the problem lies.

Do Check!!

**Comment by SinchanaMS** [comment]
Looks good to me!

**Pull request merged**